src/com/facebook/buck/features/go/CGoCompileStep.java

```diff
@@ -58,6 +58,8 @@ public CGoCompileStep(
         .add("-importpath", context.getBuildCellRootPath().toString())
         .add("-srcdir", context.getBuildCellRootPath().toString())
         .add("-objdir", outputDir.toString())
+        .add("--")
+        .add("-I" + getBuildTarget().get().getBasePath().toString())
         .addAll(cgoCompilerFlags)
```

Comment:
I think this should be before "--" otherwise the cgoCompilerFlags will be passed to the gcc

Reply:
I think it's reasonable to add extra arg to CgoLibraryDescription 'cgoCxxCompilerFlags'

Reply:
Fixed.